infrastructure/application/utils/generateTeamSecrets.ts

```diff
@@ -25,6 +25,7 @@ export const generateTeamSecrets = (
 ): awsx.ecs.KeyValuePair[] => {
   const secrets: awsx.ecs.KeyValuePair[] = [];
   teams.forEach((team) => {
+    if (env === "production" && team?.stagingOnly) return;
```

Comment:
_Comment:_

I think this is a simple change and should work although I guess we can't test it until it's on staging/prod?